### PR TITLE
updates helm charts

### PR DESCRIPTION
**Pull request by JBAhire**
The last changes,
- removes usage of `federated service` in `dev` cluster mode, for local mode, use `docker-compose` setup. 
- fixes service selector associated with `federated service` 
- fixes raw-spans-grouper's stateful set requriments

### CODE REVIEW

File: kubernetes/clusters/dev/values.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ zookeeper:
     failureThreshold: 8
 
 kafka:
-  replicaCount: 1
+  replicaCount: 3
```
Comment (kotharironak) [inline kubernetes/clusters/dev/values.yaml:26]:
```suggestion
  replicaCount: 1
```

Comment (kotharironak) [reply kubernetes/clusters/dev/values.yaml:26]: This will help in overriding - https://github.com/hypertrace/raw-spans-grouper/pull/31

**Comment by kotharironak** [comment]
The last commit updates the helm charts for raw-spans-grouper to incorporate replication factor changes needed for `dev` cluster. After this, locally verified,

**Review by ravisingal — APPROVED** [review]

**Review by adriancole — COMMENTED** [review]
great to see less overrides here

**Pull request merged**